### ISSUE TITLE
facts broken on all systems

### ISSUE THREAD

**Issue by stefanandres** [stankevich/puppet-python#184]
Since https://github.com/stankevich/puppet-python/commit/c6ca80f8200a3754dccd799a51737a5c4ace085d the facts do not do what they are supposed to do:

The pkg resources always gets the information from the python-pip package.
Beyond that, on operatingsystems != Debian/Ubuntu every puppet run will throw an ugly error, because the packages have different names than python-pip.

```
Could not retrieve fact='virtualenv_version', resolution='<anonymous>': No package found with the specified name [<unspecified category>/python-pip]
Could not retrieve fact='system_python_version', resolution='<anonymous>': No package found with the specified name [<unspecified category>/python-pip]
```

Suggeststion::
We throw that entire pkg = Code away and only keep the weight 100 facts.

I can send a PR if wanted.

**Comment by stefanandres** [comment]
I removed the broken code like this:
https://github.com/stankevich/puppet-python/compare/master...syseleven:fix-facts

**Comment by ghoneycutt** [comment]
Merged in PR #210 and released in v1.9.5

Thank you!!

**Comment by poolski** [comment]
This is still an issue in 1.10.0, unfortunately...
